include/libtorrent/string_view.hpp

```diff
@@ -41,11 +41,13 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/utility/string_ref.hpp>
 namespace libtorrent {
 using string_view = boost::string_ref;
+using wstring_view = boost::wstring_ref;
```

Comment:
btw, at this moment, only `boost.asio` is the one that requieres `boost.system` binary dependency, all the rest is using header only parts.